### PR TITLE
Documenting VPN setup

### PR DESCRIPTION
**Pull request by it33**

### CODE REVIEW

File: source/deployment/deployment.md [hunk 1]
```diff
@@ -48,15 +48,25 @@ Typically a "Mattermost unreachable" error message will be displayed warning use
...
 
-[Mattermost Push Notification Service](http://docs.mattermost.com/administration/config-settings.html?highlight=config#push-notification-settings) routes push notifications to: 
+The [Mattermost Push Notification Service (MPNS)](http://docs.mattermost.com/deployment/push.html) routes push notifications to: 
 
 1. Apple Push Notification Service to send notifications to the Mattermost iOS app. 
 2. Google Push Notification Service to send notifications to the Mattermost Android app.
 
+If you're deploying mobile applications to an Enterprise App Store, your MPNS should be behind your firewall on your private network. If you're using mobile apps in iTunes and Google Play, you can relay notifications to mobile apps using the [Hosted Push Notification Service (HPNS)](http://docs.mattermost.com/deployment/push.html#hosted-push-notifications-service-hpns) included with Mattermost Enterprise Edition. 
+
+HPNS does not connect to your mobile apps directly, it send messages over an encrypted channel to Apple or Google to relay to the apps users download from iTunes or Google Play.
```
Comment (coreyhulen) [inline source/deployment/deployment.md]: "relay to the apps users download" reads funny

**Comment by coreyhulen** [comment]
+1

**Comment by it33** [comment]
Fixed with PR: https://github.com/mattermost/docs/pull/294

File: source/deployment/deployment.md (hunk 1 above)
Comment (lfbrock) [inline source/deployment/deployment.md]: Also "it send messages" > "it sends messages"

**Pull request merged**